tests/runtime/sigusr1

```diff
@@ -0,0 +1,5 @@
+NAME print all maps
+RUN {{BPFTRACE}} -f json -e 'BEGIN { @scalar = 5; } interval:s:2 { exit(); }'
+AFTER kill -s USR1 $(pidof bpftrace)
+EXPECT @scalar[\s\S]+@scalar
+TIMEOUT 5
```

Comment:
Could you merge this file with `tests/runtime/sigint` into something like `tests/runtime/signals`? No need to have a separate suite for each signal that we handle.

Reply:
Yep, sure! I've updated the branch.